nms/app/packages/magmalte/app/components/DashboardAlertTable.js

```diff
@@ -39,52 +91,128 @@ const severityMap: {[string]: Severity} = {
   const {match} = useRouter();
   const networkId: string = nullthrows(match.params.networkId);
-
   const {isLoading, response} = useMagmaAPI(
     MagmaV1API.getNetworksByNetworkIdAlerts,
     {
       networkId,
     },
   );
-
   if (isLoading) {
     return <LoadingFiller />;
   }
 
-  const data: AlertTable = {Critical: [], Major: [], Minor: [], Other: []};
   const alerts: Array<prom_firing_alert> = response ?? [];
-  alerts.forEach(alert => {
-    const labelInfo = {
-      job: alert.labels['job'] || '',
-      instance: alert.labels['instance'] || '',
-    };
-
-    const timingInfo = {
-      startsAt: alert.startsAt || '',
-      endsAt: alert.endsAt || '',
-      updatedAt: alert.updatedAt || '',
-    };
+  const data: AlertTable = {Critical: [], Major: [], Minor: [], Other: []};
 
+  alerts.forEach(alert => {
     const sev: Severity = severityMap[alert.labels['severity']] || 'Other';
-
     data[sev].push({
-      name: alert.labels['alertname'],
-      cols: [
-        JSON.stringify(labelInfo),
-        JSON.stringify(alert.annotations),
-        JSON.stringify(alert.status),
-        JSON.stringify(timingInfo),
-      ],
+      label: alert.labels.alertname,
+      labelInfo: `${alert.labels.job} - ${alert.labels.instance}`,
+      annotations: `${alert.annotations.description} - ${alert.annotations.summary}`,
+      status: alert.status.state,
+      timingInfo: new Date(alert.startsAt),
     });
   });
-
   return (
     <>
       <CardTitleRow icon={Alarm} label={`Alerts (${alerts.length})`} />
-      <TabbedTable data={data} />
+      <AlertsTabbedTable alerts={data} />
+    </>
+  );
+}
+type TabPanelProps = {
+  alerts: Array<AlertRowType>,
+  label: string,
+};
+
+function TabPanel(props: TabPanelProps) {
+  const classes = useStyles();
+  const [_, setCurrRow] = useState<AlertRowType>({});
+  if (props.alerts.length === 0) {
+    return (
+      <Paper elevation={0}>
+        <Grid
+          container
+          alignItems="center"
+          justify="center"
+          className={classes.emptyTable}>
+          <Grid item xs={12} className={classes.emptyTableContent}>
+            <Text variant="body2">You have 0 {props.label} Alerts</Text>
+            <Text variant="body3">
+              To add alert triggers click <a href="#">alert settings</a>.
+            </Text>
+          </Grid>
+        </Grid>
+      </Paper>
+    );
+  }
+
+  return (
+    <ActionTable
+      data={props.alerts}
+      columns={[
+        {title: 'Label', field: 'label'},
+        {title: 'Label Info', field: 'labelInfo'},
+        {title: 'Annotations', field: 'annotations'},
+        {title: 'Status', field: 'status'},
+        {title: 'Date', field: 'timingInfo', type: 'datetime'},
+      ]}
+      handleCurrRow={(row: AlertRowType) => setCurrRow(row)}
+      options={{
+        actionsColumnIndex: -1,
+        pageSizeOptions: [5, 10],
+        toolbar: false,
+        header: false,
+      }}
+      actionHeader={false}
```

Comment:
You might want to make this localization={
actionHeader:false
}. I added localization prop earlier as I mentioned over chat